randovania/games/super_metroid/json_data/Wrecked Ship.txt

```diff
@@ -320,7 +312,11 @@ Extra - asset_id: 63
   * Layers: default
   * Normal Door to East Ocean/Door to Electric Death Room
   > Door to Spiky Death Room
-      Gravity Suit or Hi-Jump Boots or Space Jump
+      Any of the following:
+          # Jump out of water
```

Comment:
Not sure if this should be its own trick, but you can look down while jumping and Samus' hitbox gets a little smaller allowing here to get onto ledges just barely out of reach. You can get out of the water here with nothing. 

https://youtu.be/Znv9j_X02vM

Additionally, if the power isn't on this connection is trivial.

Reply:
why is it trivial if power is off? is the water not there?

Reply:
Correct, there is not water in these rooms with the power off.